packages/blade-old/package.json

```
@@ -25,7 +28,8 @@
     "test:native": "BABEL_ENV=development jest -c ./jest.native.config.js",
     "test:native:coverage": "yarn run test:native --coverage",
     "test:web": "BABEL_ENV=web-test jest -c ./jest.web.config.js",
-    "test:web:coverage": "yarn run test:web --coverage"
+    "test:web:coverage": "yarn run test:web --coverage",
+    "postinstall": "cd ./scripts/icon-migrate-codemod && npm install"
```

```suggestion
    "postinstall": "cd ./scripts/icon-migrate-codemod && yarn"
```

also remove package-lock